pages/product.js

```diff
@@ -0,0 +1,31 @@
+const { I } = inject();
+
+module.exports = {
+  colorDropDown: { xpath: '//label[text()="Color"]/following-sibling::div/a[1]' },
+  sizeDropDown: { xpath: '//label[text()="Size"]/following-sibling::div/a[1]' },
+  colorOption: { xpath: '//label[text()="Color"]/following-sibling::div/ul/li[2]' },
+  sizeOption: { xpath: '//label[text()="Size"]/following-sibling::div/ul/li[2]' },
+  productPriceText: { xpath: '//div/span[@class="price-new"]' },
+
+  selectColor() {
+    I.click(this.colorDropDown);
+    I.click(this.colorOption);
+  },
+
+  selectSize() {
+    I.click(this.sizeDropDown);
+    I.click(this.sizeOption);
+  },
+
+  async getProductPrice() {
+    const draftProductPrice = await I.grabTextFrom(this.productPriceText);
+    const draftPrice = parsePrice(this.draftProductPrice);
```

Comment:
`this.draftProductPrice` - you are referring to object's key `draftProductPrice`, not variable `draftProductPrice`